amundsen_application/static/js/components/TableDetail/OwnerEditor/index.tsx

```diff
@@ -25,7 +25,6 @@ export interface DispatchFromProps {
 
 export interface ComponentProps {
   errorText?: string | null;
-  readOnly: boolean;
```

Comment:
We have a check below somewhere for `!this.state.readOnly` that should be removed since it looks like we're no longer using this at all and it will always evaluate to `true` with out current logic. Can we fully clean up the usage of `readOnly` in this file?

Reply:
`state.readOnly` gets set to true when an owner update request fails. I'm opting to maintain this behavior since this PR is supposed to just upgrade libraries.